src/hotspot/share/cds/aotCacheAccess.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Shouldn't the copyright year be just 2025? Same for other new files.

Reply:
Right. I will fix that.